[debug] [2020-05-08T05:12:48.933Z] ----------------------------------------------------------------------
[debug] [2020-05-08T05:12:48.936Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\venkata.mannava\AppData\Roaming\npm\node_modules\firebase-tools\lib\bin\firebase.js deploy
[debug] [2020-05-08T05:12:48.936Z] CLI Version:   8.2.0
[debug] [2020-05-08T05:12:48.936Z] Platform:      win32
[debug] [2020-05-08T05:12:48.936Z] Node Version:  v12.13.0
[debug] [2020-05-08T05:12:48.960Z] Time:          Fri May 08 2020 10:42:48 GMT+0530 (India Standard Time)
[debug] [2020-05-08T05:12:48.961Z] ----------------------------------------------------------------------
[debug] [2020-05-08T05:12:48.961Z] 
[debug] [2020-05-08T05:12:48.994Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2020-05-08T05:12:48.995Z] > authorizing via signed-in user
[debug] [2020-05-08T05:12:48.996Z] [iam] checking project expenditure-tracker-malathi for permissions ["firebase.projects.get","firebasehosting.sites.update"]
[debug] [2020-05-08T05:12:48.999Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2020-05-08T05:12:48.999Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  
 <request body omitted>
[debug] [2020-05-08T05:12:49.418Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=utf-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 08 May 2020 05:12:51 GMT","server":"scaffolding on HTTPServer2","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-05-08T05:12:49.424Z] >>> HTTP REQUEST POST https://cloudresourcemanager.googleapis.com/v1/projects/expenditure-tracker-malathi:testIamPermissions  
 {"permissions":["firebase.projects.get","firebasehosting.sites.update"]}
[debug] [2020-05-08T05:12:51.262Z] <<< HTTP RESPONSE 200 {"content-type":"application/json; charset=UTF-8","vary":"X-Origin, Referer, Origin,Accept-Encoding","date":"Fri, 08 May 2020 05:12:52 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","server-timing":"gfet4t7; dur=1495","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-05-08T05:12:51.264Z] [iam] error while checking permissions, command may fail: FirebaseError: Authorization failed. This account is missing the following required permissions on project expenditure-tracker-malathi:

  firebase.projects.get
  firebasehosting.sites.update
[debug] [2020-05-08T05:12:51.266Z] >>> HTTP REQUEST GET https://firebase.googleapis.com/v1beta1/projects/expenditure-tracker-malathi  
 
[debug] [2020-05-08T05:12:51.973Z] <<< HTTP RESPONSE 403 {"vary":"X-Origin, Referer, Origin,Accept-Encoding","content-type":"application/json; charset=UTF-8","date":"Fri, 08 May 2020 05:12:53 GMT","server":"ESF","cache-control":"private","x-xss-protection":"0","x-frame-options":"SAMEORIGIN","x-content-type-options":"nosniff","alt-svc":"h3-27=\":443\"; ma=2592000,h3-25=\":443\"; ma=2592000,h3-Q050=\":443\"; ma=2592000,h3-Q049=\":443\"; ma=2592000,h3-Q048=\":443\"; ma=2592000,h3-Q046=\":443\"; ma=2592000,h3-Q043=\":443\"; ma=2592000,quic=\":443\"; ma=2592000; v=\"46,43\"","accept-ranges":"none","transfer-encoding":"chunked"}
[debug] [2020-05-08T05:12:51.974Z] <<< HTTP RESPONSE BODY {"error":{"code":403,"message":"The caller does not have permission","status":"PERMISSION_DENIED"}}
[debug] [2020-05-08T05:12:51.976Z] HTTP Error: 403, The caller does not have permission
[debug] [2020-05-08T05:12:51.982Z] FirebaseError: HTTP Error: 403, The caller does not have permission
    at module.exports (C:\Users\venkata.mannava\AppData\Roaming\npm\node_modules\firebase-tools\lib\responseToError.js:38:12)
    at Request._callback (C:\Users\venkata.mannava\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:41:35)
    at Request.self.callback (C:\Users\venkata.mannava\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at Request.emit (events.js:210:5)
    at Request.EventEmitter.emit (domain.js:476:20)
    at Request.<anonymous> (C:\Users\venkata.mannava\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1154:10)
    at Request.emit (events.js:210:5)
    at Request.EventEmitter.emit (domain.js:476:20)
    at IncomingMessage.<anonymous> (C:\Users\venkata.mannava\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:1076:12)
    at Object.onceWrapper (events.js:299:28)
    at IncomingMessage.emit (events.js:215:7)
    at IncomingMessage.EventEmitter.emit (domain.js:476:20)
    at endReadableNT (_stream_readable.js:1183:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
[error] 
[error] Error: Failed to get Firebase project expenditure-tracker-malathi. Please make sure the project exists and your account has permission to access it.
